commit d16cd0b5926db0b3e522f2a2190eda5fcc4b211f
Author: Todd <toddrme2178@gmail.com>
Date:   Fri Aug 10 14:11:08 2018 -0400

    Include test file in sdists
    
    Downstream packagers often use the tests to make sure the packages are configured and built correctly.

diff --git a/MANIFEST.in b/MANIFEST.in
index 1aba38f..750b9cb 100644
--- a/MANIFEST.in
+++ b/MANIFEST.in
@@ -1 +1,3 @@
 include LICENSE
+include ci.py
+recursive-include test *
